scripts/components/api-changes-validator/api_usage_statements_generators.ts

```diff
@@ -230,6 +228,14 @@ export class ClassUsageStatementsGenerator implements UsageStatementsGenerator {
       }
     }
 
+    if (this.classDeclaration.heritageClauses) {
```

Comment:
I suppose this is technically correct usage of "heritage", but I would just say "inheritance" wherever the word "heritage" is currently used in the PR

Reply:
Changed where I could. Some remained as it's part of TS compiler api.